### PR TITLE
[GE] ImagebasedSlider, icon additions. and css improvements

### PR DESCRIPTION
**Pull request by msDestiny14**
image

https://github.com/BabylonJS/ThePirateCove/issues/92

### CODE REVIEW

File: sandbox/src/components/dropUpButton.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ export class DropUpButton extends React.Component<IDropUpButtonProps, { isOpen:
             <div className="dropup">
                 {this.props.icon && (
                     <div className={"button" + (this.state.isOpen ? " active" : "")} onClick={() => this.switchDropUp()}>
-                        <img src={this.props.icon} title={this.props.label} alt={this.props.label} />
+                        <img src={this.props.icon} width={"30px"} title={this.props.label} alt={this.props.label} />
```
Comment (sebavan) [inline sandbox/src/components/dropUpButton.tsx:57]: same width comment about moving to css as it seems to be used in more than one place making probably even a better candidate for css.

File: sandbox/src/components/footerButton.tsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ export class FooterButton extends React.Component<IFooterButtonProps> {
 
         return (
             <div className="button" onClick={() => this.props.onClick()}>
-                <img src={this.props.icon} alt={this.props.label} title={this.props.label} />
+                <img src={this.props.icon} width={"30px"} alt={this.props.label} title={this.props.label} />
```
Comment (sebavan) [inline sandbox/src/components/footerButton.tsx:20]: same

File: sandbox/src/components/footerFileButton.tsx [hunk 3]
```diff
@@ -21,7 +21,7 @@ export class FooterFileButton extends React.Component<IFooterFileButtonProps> {
 
         return (
             <div className="custom-upload" title={this.props.label}>
-                <img src={this.props.icon} />
+                <img src={this.props.icon} width={"30px"} />
```
Comment (sebavan) [inline sandbox/src/components/footerFileButton.tsx:24]: same

**Review by sebavan — COMMENTED** [review]
mostly nits and a shared css for width 30, will let @RaananW do the final validation

**Review by RaananW — COMMENTED** [review]
LGTM, please address @sebavan comments :-)

**Pull request merged**